pkg/service/audit_utils.go

```diff
@@ -112,6 +112,25 @@ func CreateUserAuditEvent(ctx context.Context, al *zap.Logger, db bun.IDB, actio
 	}
 }
 
+func CreateUserLoginAuditEvent(ctx context.Context, al *zap.Logger, action string, name string) {
+	sd, ok := GetSessionDataFromContext(ctx)
```

Comment:
There is chance that session data is not set because this request is excluded from authentication. @mabhi Can you check if `actor` and `client` information is populated in the event?

Reply:
```
/audit-logs # cat audit.log 
{"timestamp":"2022-12-12T07:35:15.201213382Z","version":"1.0","category":"AUDIT","origin":"core","actor":{"type":"USER","account":{"username":"some-two@user.com"},"groups":null},"client":{"type":"BROWSER","ip":"","user_agent":"","host":""},"detail":{"message":"User login: some-two@user.com","meta":{"user":"some-two@user.com"}},"type":"user.login.success","portal":"OPS","project":""}

````
This is from inside the paralus container. Do I have to add something more or information missing ?
As this method `CreateUserLoginAuditEvent` is getting called from `service.CreateLoginAuditLog` in user.go the session data is always populated with atleast _Username_. This is guarded against blank / no usernames.

> new_ctx := context.WithValue(ctx, common.SessionDataKey, &commonv3.SessionData{Username: username})
> CreateUserLoginAuditEvent(new_ctx, s.al, "login", username)

Reply:
this is being tracked under - https://github.com/paralus/paralus/issues/119